models/TokensModel.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs').promises;
+
+class TokensModel {
+
+    constructor(obj){
+        fs.readFile("tokens.json", "utf-8").then(tokensJSON => {
+            this.object = JSON.parse(tokensJSON)
+            this.array = this.object.tokens;
+        })
+    }
+
+    async add(email, refreshToken) {
+        this.array.push({email, refreshToken});
+        await this.save();
+    }
+
+    isExists(email){
```

Comment:
`is exists` is incorrect statement in English. Simplifying to `exists` will work fine I think.